en/tutorials-and-examples/blog/part-two.rst

```diff
@@ -418,9 +418,9 @@ back at our Articles model and make a few adjustments::
         public function validationDefault(Validator $validator): Validator
         {
             $validator
-                ->notEmpty('title')
+                ->notEmptyString('title')
                 ->requirePresence('title')
```

Comment:
also requirePresence without 2nd param `'create'` is usually an anti pattern and corrupts partial updates.